docker/Dockerfile

```diff
@@ -8,15 +8,16 @@ RUN yum -y update &&\
     yum -y install curl &&\
     adduser compass_user
 
-COPY . /home/compass_user/OPERA/COMPASS
+RUN mkdir -p /home/compass_user/OPERA
```

Comment:
Not a critical issue, we can work on that when preparing for the next release.
Maybe we can tie this RUN command with the following one, so that we can reduce the layer of the docker image.